footprint: [[0.0,0.18],[-0.18,0.18],[-0.18,-0.18],[0.0,-0.18],[0.17,-0.18],[0.25,0.0],[0.17,0.18]]

transform_tolerance: 1.0
# update_frequency: 3.0
# publish_frequency: 3.0

#must match incoming static map
global_frame: /map
robot_base_frame: /base_footprint
resolution: 0.05

# rolling_window: false
# track_unknown_space: true

plugins:
- {name: static_layer,            type: "costmap_2d::StaticLayer"}   # for global map
- {name: explore_boundary,  type: "frontier_exploration::BoundedExploreLayer"}
#Can disable sensor layer if gmapping is fast enough to update scans
- {name: obstacle_layer,            type: "costmap_2d::ObstacleLayer"}
- {name: inflation_layer,         type: "costmap_2d::InflationLayer"}

explore_boundary:
  resize_to_boundary: false
  frontier_travel_point: middle
  explore_clear_space: false
